[FR] Guides (#2417)
ppy/osu-wiki: wiki/Guides/fr.md

# Guides

> Vous trouverez ci-dessous une liste triée des guides créés par des membres de la communauté osu! ils proviennent des forums osu! et ont été transférées sur le osu! wiki.

Reviewer: ```suggestion
Vous trouverez ci-dessous une liste triée des guides créés par des membres de la communauté osu! dont la plupart proviennent des forums osu! et ont été transférés sur ce wiki.
```

> Vous trouverez ci-dessous une liste triée des guides créés par des membres de la communauté osu! la plupart proviennent des forums osu! et ont été transférés sur ce wiki.

Reviewer: ```suggestion
Vous trouverez ci-dessous une liste triée des guides créés par des membres de la communauté osu! dont la plupart proviennent des forums osu! et ont été transférés sur ce wiki.
```

Visiblement j'ai du mal à faire des suggestions complètes.

## Beatmapping

*Page principale : [Beatmapping](/wiki/Beatmapping)*

- [Ajouter des sons personalisés](/wiki/Adding_Custom_Hit_Sounds)
- [Édition audio](/wiki/Audio_Editing)
- [Changer l'artiste ou le titre](/wiki/Changing_the_Artist_or_Title)
- [Changer le texte du titre](/wiki/Changing_the_Title_Text)

> - [Information sur les collaborations](/wiki/Collab_Information)

Reviewer: ```suggestion
- [Informations sur les collaborations](/wiki/Collab_Information)
```

> - [Fichier compressé](/wiki/Compressing_Files)

Reviewer: ```suggestion
- [Compresser des fichiers](/wiki/Compressing_Files)
```

> - [Librairie de son personalisés](/wiki/Custom_Hit_Sound_Library)

Reviewer: ```suggestion
- [Bibliothèque de sons personalisés](/wiki/Custom_Hit_Sound_Library)
```

Je trouve `Bibliothèque` plus approprié dans ce cas, mais c'est peut être juste moi.

Author: Effectivement, je trouve ça plus joli aussi.

> - [Remplacements d'échantillons personnalisés](/wiki/Custom_Sample_Overrides)

Reviewer: ```suggestion
- [Utiliser des échantillons personnalisés](/wiki/Custom_Sample_Overrides)
```

Autant essayer d'être cohérent et utiliser l'infinitif quand il y a un verbe.

- [Comment obtenir des chansons de jeux vidéo](/wiki/Getting_Songs_From_Video_Games)
- [Comment timer les musiques](/wiki/How_to_Time_Songs)
- [Théorie de la musique](/wiki/Music_Theory)
- [Guide pour mapper sur osu!mania](/wiki/osu!mania_Mapping_Guide)

> - [Réglage du décalage sur le temps correct](/wiki/Setting_the_Offset_on_the_Correct_Beat)

Reviewer: ```suggestion
- [Régler le décalage sur le bon temps](/wiki/Setting_the_Offset_on_the_Correct_Beat)
```

> - [Commencez un projet de beatmap](/wiki/Starting_a_Beatmap_Project)

Reviewer: ```suggestion
- [Commencer un projet de beatmap](/wiki/Starting_a_Beatmap_Project)
```

- [Vidéos venant de YouTube](/wiki/Videos_From_Youtube)

## Modding

*Page principale : [Modding](/wiki/Modding)*

> - [Se faire modder sa map](/wiki/Getting_Your_Map_Modded)

Reviewer: Je ne sais pas si on a besoin du `Se`, ça pourrait juste être

```suggestion
- [Faire modder sa map](/wiki/Getting_Your_Map_Modded)
```

- [Comment faire ranker votre map](/wiki/How_To_Get_Your_Map_Ranked)

> ## Comment jouer

Reviewer: ```suggestion
## Jouer
```

Comme ça tous les titres ont le même modèle.

- [Tutoriel du débutant](/wiki/Beginner's_Tutorial)
- [Aide aux problèmes d'incompatibilité d'OpenGL](/wiki/GL_Support_Issues)
- [Comment jouer à osu!mania](/wiki/How_to_Play_osu!mania)

> - [Comment utiliser l'assistant offset](/wiki/How_to_Use_the_Offset_Wizard)

Reviewer: Pourquoi garder `offset` ici alors que tu utilises `décalage` plus haut ?

> - [Faire des enregistrement d'osu!](/wiki/Recording_osu!)

Reviewer: ```suggestion
- [Faire des enregistrements d'osu!](/wiki/Recording_osu!)
```

- [Chercher et télécharger des beatmaps](/wiki/Searching_and_Downloading_Beatmaps)

> - [Achat de tablette](/wiki/Tablet_Purchase)

Reviewer: Cette tournure me paraît un peu étrange, je suggère:

```suggestion
- [Acheter une tablette](/wiki/Tablet_Purchase)
```

- [Trucs et astuces pour améliorer ses compétences](/wiki/Tips_and_Tricks_on_Skill_Improvement)

## Skinning

*Page principale : [Skinning](/wiki/Skinning)*

*Voir aussi: [Tutoriel de Skinning](/wiki/Skinning_Tutorial)*

> - [Recadrer des arrière-plan complexes](/wiki/Cropping_with_Complex_Backgrounds)

Reviewer: ```suggestion
- [Recadrer des arrière-plans complexes](/wiki/Cropping_with_Complex_Backgrounds)
```

Je ne suis pas sûr que `recadrer` soit correct pour ce guide mais je ne vois pas mieux, de même pour le guide suivant.

> - [Recadrer des arrière-plan simples](/wiki/Cropping_with_Simple_Backgrounds)

Reviewer: ```suggestion
- [Recadrer des arrière-plans simples](/wiki/Cropping_with_Simple_Backgrounds)
```

> - [Faire des Spinners proprement centrés](/wiki/Making_Properly_Centered_Spinners)

Reviewer: Est-ce qu'il y a besoin d'une majuscule à `Spinner` ?
